inputfiles/overridingTypes.jsonc

```diff
@@ -1,6 +1,14 @@
 {
     "mixins": {
         "mixin": {
+            "ChildNode": {
+                "name": "ChildNode",
+                "extends": "Node"
```

Comment:
TBH https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1087#issuecomment-891333791 is better to me with more spec compliance, why is this better?

Reply:
BTW, in this way `name` is redundant.

Reply:
I mentioned it in https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1087#issuecomment-891428070, but I don't think the `ChildNodeBase` and `ParentNodeBase` are strictly necessary and I'd rather be conservative about additions so close the RC.

Reply:
>so close the RC.

Fair enough 😄